prometheus-to-sd/translator/metric_descriptor_cache.go

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
stackdriver.go file was from the beginning in translator package.

But I have moved all stackdriver calls to the corresponding file.